src/main/resources/mockdata/1ab7ef6a-d1d4-4a4f-90a2-882aed18af14.json

```diff
@@ -1,7 +1,154 @@
 {
   "purchase_order": {
     "id": "1ab7ef6a-d1d4-4a4f-90a2-882aed18af14",
-    "po_number": "268758"
+    "po_number": "268758",
+    "created": "2018-06-29T03:00:00.000+0000",
+    "comments": "",
+    "total_items": 3,
+    "total_estimated_price": 68.00,
+    "adjustments": "",
+    "assigned_to": "Ann",
+    "created_by": "mu44",
+    "transmission_date": "",
+    "transmission_method": "",
+    "po_workflow_status_id": "Open",
+    "po_receipt_status": "not received",
+    "po_payment_status": ""
   },
-  "po_lines": []
+  "po_lines": [
+    {
+      "id": "",
```

Comment:
just assign a random UUID.  I use https://www.uuidgenerator.net/ for things like this.

Reply:
This goes for all of the "id" fields